tests_e2e/tests/agent_not_provisioned/disable_agent_provisioning.py

```diff
@@ -62,31 +55,9 @@ def update(self, template: Dict[str, Any], is_lisa_template: bool) -> None:
         #         }
         #     }
         #
-        # The code below sets template['functions'][i]['members']['getLinuxConfiguration']['output']['value']['provisionVMAgent'] to True,
```

Comment:
This code got refactored to the base class (UpdateArmTemplate) since now it is used by multiple tests